arch/arm/src/stm32wl5/kconfig.d/configuration.kconfig

```diff
@@ -0,0 +1,4 @@
+source "arch/arm/src/stm32wl5/kconfig.d/configuration/serial.kconfig"
```

Comment:
why create kconfig.d/configuration/ too?

Reply:
No particular reason I guess, just felt right to me. I guess maybe config and periph dirs are not really needed, as I don't think there will be anything else there apart from periph and configuration. So periphs could go to one file, as it's mostly flat, and rest of the files (configurations) could be kept at kconfig.d/ root. What do you say?

Reply:
See the above reply.